src/changelog.ts

```diff
@@ -139,20 +139,21 @@ function stringifyLinkReferenceDefinitions(
   // patch releases on older releases can be accomodated.
   const releaseLinkReferenceDefinitions = releases
     .map(({ version }) => {
+      let diffUrl;
       if (version === orderedReleases[orderedReleases.length - 1]) {
```

Comment:
My only feedback is that `orderedReleases` (and really the releases param) could be documented / named such that it's immediately obvious that the releases are sorted in chronological order.k

Reply:
Good call, I'll rename this

Reply:
Done in https://github.com/MetaMask/auto-changelog/pull/76/commits/027e394c92bc254503aa153ad009ea6541f57282